sdk/storage/azure-storage-file-datalake/tests/test_large_file.py

```diff
@@ -80,8 +80,11 @@ def test_append_large_stream_without_network(self):
         self.assertEqual(self.payload_dropping_policy.append_counter, 1)
         self.assertEqual(self.payload_dropping_policy.append_sizes[0], LARGEST_BLOCK_SIZE)
 
-    @pytest.mark.live_test_only
+    @record
     def test_upload_large_stream_without_network(self):
+        if not TestMode.is_playback(self.test_mode):
```

Comment:
this test failed with Linux Pypy3 on this line:
`self.assertEqual(self.payload_dropping_policy.append_counter, 2)`
append_counter is 1 somehow. I'm still setting up environment on Ubuntu to reproduce the problem